extensions/cloud/portability-cloud-google/src/main/java/org/datatransferproject/cloud/google/GoogleJobStore.java

```diff
@@ -338,6 +339,46 @@ public void create(UUID jobId, String key, InputStream stream) {
     googleTempFileStore.create(jobId, key, stream);
   }
 
+  @Override
+  public void addCounts(UUID jobId, Map<String, Integer> newCounts) throws IOException {
+    if (newCounts == null) {
```

Comment:
Do we just want to make this a precondition or are we expecting null?

Reply:
The localjobstore implementation expects null sometimes >.> 
And we default to null here: https://github.com/google/data-transfer-project/blob/master/portability-transfer/src/main/java/org/datatransferproject/transfer/PortabilityInMemoryDataCopier.java#L164

We could change that to be an empty map instead